cookbooks/swift/files/default/etc/prometheus/prometheus-rules.yaml

```diff
@@ -0,0 +1,5 @@
+groups:
+  - name: Cluster aggregates
+    rules:
+      - record: object_server:sum
+        expr: sum (object_server) by (method)
```

Comment:
I'm not sure this is working properly yet :confused:

Long term, though, the idea is to have a place where we can document how to aggregate cluster-wide metrics.

Reply:
`prometheus.yaml` had a mistake -- I'm a little surprised it didn't fail to start.